parser.py

```diff
@@ -1,6 +1,22 @@
 from bs4 import BeautifulSoup
+import bs4
 from typing import Generator, Union, Optional, List
 
+soup_class_dict = {'job_name': '_3862j6',
+                   'job_url': '_15V35X',
+                   'salary_tag': 'jNebTl',
+                   'short_description': '_9jGwm1',
+                   'full_description_raw': '_1yTVFy'}
+
+
+def soup_find_exception_checker(line: Union[str, bs4.element.ResultSet, bs4.element.Tag]):  # typing ?
```

Comment:
I suppose it should return `str`, or `Optional[str]`. If you choose first variant then let's return empty string `""` but not  `'empty field or wrong value'`